debian/org.openlighting.ola.ola.metainfo.xml

```diff
@@ -27,6 +27,8 @@
     <modalias>usb:v0962p*</modalias> <!-- USBDMX2 -->
     <modalias>usb:v10CFp8062d*</modalias> <!-- Velleman -->
     <modalias>usb:v16D0p0830d*</modalias> <!-- DMXControl Projects e.V. Nodle U1 -->
```

Comment:
While you're at it, would you mind adding the *DMXCreator 512 Basic* USB device here too? I didn't know about this file (and actually don't know what it is for now) when I added support for that adapter (see #1136). So probably just one line:

```xml
<modalias>usb:v0a30p0002d*</modalias> <!-- DMXCreator 512 Basic -->
```

(for vendor ID `0a30`, product ID `0002`; I used the same syntax as everywhere else)

Reply:
sure

Reply:
I probably forgot to mention it @FloEdelmann . It came up on Lintian here https://lintian.debian.org/tags/appstream-metadata-missing-modalias-provide.html the theory is when you plug in a device your machine can suggest which software to use to control it, rather than you having to guess/look it up. I need @yoe to build a new deb for 0.10.6 when he gets a chance to confirm I've cleared the lint issues, but I think it should be fine now.

Reply:
This has been done elsewhere.